llvm/lib/ExecutionEngine/JITLink/aarch32.cpp

```diff
@@ -20,6 +20,8 @@
 #include "llvm/Support/ManagedStatic.h"
 #include "llvm/Support/MathExtras.h"
 
+#include "JITLinkAArch32.inc"
+
```

Comment:
I initially designed the backend to be Edge Kind dependent and for this reason I could not include the generated file in the header due to circular dependency issues. A Macro based design will solve this issue as the interpretation will be done in header instead of the backend.